build.gradle.kts

```diff
@@ -10,7 +11,7 @@ configurations {
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springframework.boot:spring-boot-starter-jdbc")
-
+  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
```

Comment:
If we add the data-jpa we can remove the jdbc one. I do not think that at this stage JPA level is needed , and we should stick with starter-data-jdbc.